CMakeLists.txt

```diff
@@ -1170,7 +1170,11 @@ if(NOT USE_CUDA AND NOT USE_ROCM)
 endif()
 
 if(BUILD_NVFUSER)
-  add_subdirectory(third_party/nvfuser)
+  if(DEFINED ENV{NVFUSER_SOURCE_DIR})
+    add_subdirectory($ENV{NVFUSER_SOURCE_DIR} nvfuser)
+  else()
+    add_subdirectory(third_party/nvfuser nvfuser)
+  endif()
```

Comment:
Is this a temporary workaround?

Reply:
Yes!

Before we get our submodule setup properly, we would hope to stick with upstream pytorch code base `https://github.com/pytorch/pytorch/` while using our development nvfuser branch for our internal nightly container.